storybook/manager.js

```diff
@@ -1,15 +1,18 @@
 /**
  * External dependencies
  */
-import { addons } from '@storybook/manager-api';
+import { addons } from 'storybook/manager-api';
+import { defaultConfig } from 'storybook-addon-tag-badges';
 
 /**
  * Internal dependencies
  */
-import theme from './theme';
+import { tagBadges } from './badges';
 import sidebar from './sidebar';
+import theme from './theme';
 
 addons.setConfig( {
 	sidebar,
+	tagBadges: [ ...tagBadges, ...defaultConfig ],
```

Comment:
I don't think we'll need the default config.

Reply:
Updated.